recipes/graphene/all/conanfile.py

```diff
@@ -51,6 +52,7 @@ def layout(self):
 
     def requirements(self):
         if self.options.with_glib:
+            # leave run=can_run(self) on requires for compatibility reasons.
             self.requires("glib/2.75.2")
```

Comment:
You probably want the same version specified in both the `requires` and `tool_requires`.

```suggestion
            self.requires("glib/2.76.0")
```